docs/pages/machine-id/access-guides/ssh.mdx

```diff
@@ -3,4 +3,155 @@ title: Machine ID with Server Access
+can be used to grant machines secure, short-lived access to these servers.
+
+In this guide, you will configure `tbot` to produce credentials that can be
+used to access a Linux server enrolled in Teleport Server Access. The guide
+will cover access using Teleports CLI `tsh` as well as the OpenSSH SSH client.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- If you have not already connected your server to Teleport, follow
+  the [Server Access Getting Started Guide](../../application-access/getting-started.mdx).
+
+- (!docs/pages/includes/tctl.mdx!)
+
+- `tbot` must already be installed and configured on the machine that will
+  connect to Linux hosts with SSH. For more information, see the
+  [platform guides](../platform-guides.mdx).
+
+## Step 1/3. Configure RBAC
+
+First, Teleport must be configured to allow the credentials produced by the bot
+to connect to the SSH servers. In this example, access will be granted to all
+SSH nodes for the username `root`.
+
+For production use, you should use labels to restrict this access to only the
+hosts that the bot will need to access. This is known as the principal of
+least privilege and limits the damage that exfiltrated credentials can do.
+
+You may also wish to adjust this to grant access to a user other than `root`
+depending on what commands will need to be executed with the credentials.
+
+If you have followed an access guide, you will have created a role and granted
+the bot the ability to impersonate it already. This role just needs to have the
+additional privileges added to it.
+
+Use `tctl edit role/example-bot` to add the following to the role:
+
+```yaml
+spec:
+  allow:
+    # Allow login to the Linux user 'root'.
+    logins: ['root']
+    # Allow connection to any node. Adjust these labels to match only nodes
+    # that ansible needs to access.
+    node_labels:
+      '*': '*'
+```
+
+## Step 2/3. Configure the `tbot` output
+
+Now, `tbot` needs to be configured with an output that will produce the
+SSH certificate and OpenSSH configuration. For this we use the `identity` output
+type.
+
+Outputs must be configured with a destination. In this example, the `directory`
+destination will be used. This will write these credentials to a specified
+directory on disk. Ensure that this directory can be written to by the Linux
+user that `tbot` runs as, and that it can be read by the Linux user that needs
+to use SSH.
+
+Modify your `tbot` configuration to add an `identity` output:
+
+```yaml
+outputs:
+- type: identity
+  destination:
+    type: directory
+    # For this guide, /opt/machine-id is used as the destination directory.
+    # You may wish to customize this. Multiple outputs cannot share the same
+    # destination.
+    path: /opt/machine-id
+```
+
+If operating `tbot` as a background service, restart it. If running `tbot` in
+one-shot mode, it must be executed before you attempt to execute the Ansible
+playbook.
```

Comment:
```suggestion
one-shot mode, it must be executed before you attempt to connect your machine to a server registered with Teleport.
```
Or something else that fixes the miscopied text.

Reply:
11f7830f93